### DISCUSSION TITLE
Scaling Combat Mission Games / Games That Do Not Have Windowed Mode

### DISCUSSION THREAD

**Discussion by TooSpooky** [Blinue/Magpie#314] [Q&A]
Hello,

I am wondering if there is some possible way to scale Combat Mission games using Magpie, which do not have a windowed mode as far as I'm aware. I've tried a few dx applications to force the games into windowed but they did not work, although I could be using them incorrectly.

I'd appreciate any insight or help, especially since they have seemingly 0 UI scale and poor performance so 4k native isn't a joy haha.

Thanks and take care.

**Comment by Blinue** [comment]
You'll have to find a way to window the game, as Magpie does not work in fullscreen games.

**Comment by Tzugimaa** [comment]
One of the primitive ways I can think of is Alt + Enter.
If that does not work try : https://help.steampowered.com/en/faqs/view/7D01-D2DD-D75E-2955 
1. Right click at the game.
2. Select "Properties...".
3. In "LAUNCH OPTIONS" type in "-windowed".

Hope it helps
